iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/TieLineUtil.java

```diff
@@ -77,28 +84,37 @@ public static void mergeProperties(DanglingLine dl1, DanglingLine dl2, Propertie
                 properties.setProperty(prop, dl1.getProperty(prop));
             } else if (dl1.getProperty(prop).isEmpty()) {
                 LOGGER.debug("Inconsistencies of property '{}' between both sides of merged line. Side 1 is empty, keeping side 2 value '{}'", prop, dl2.getProperty(prop));
+                propertyOnlyOnOneSide(reporter, prop, dl2.getProperty(prop), 1, dl1.getId(), dl2.getId());
                 properties.setProperty(prop, dl2.getProperty(prop));
             } else if (dl2.getProperty(prop).isEmpty()) {
                 LOGGER.debug("Inconsistencies of property '{}' between both sides of merged line. Side 2 is empty, keeping side 1 value '{}'", prop, dl1.getProperty(prop));
+                propertyOnlyOnOneSide(reporter, prop, dl1.getProperty(prop), 2, dl1.getId(), dl2.getId());
```

Comment:
Shouldn't it be:
```suggestion
                propertyOnlyOnOneSide(reporter, prop, dl1.getProperty(prop), 2, dl2.getId(), dl1.getId());
```

Reply:
No it should be always side 1 first then side 2